build.ps1

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env pwsh
+#Requires -Version 5.1
+
+<#
+.SYNOPSIS
+  Build Project.
+.DESCRIPTION
+  You can used this script to make publish packages for your game.
+.PARAMETER Games
+  What game would you want to build.
+  Build all games when they empty.
+.PARAMETER IsDebug
+  Build projects by debug mode.
```

Comment:
when they empty -> if empty / if omitted
by debug mode -> in debug mode

Reply:
```suggestion
.PARAMETER Games
  What game would you want to build.
  Build all games by default.
.PARAMETER IsDebug
  Build projects in debug mode.
```